infrastructure/clients/youtube_v3.go

```diff
@@ -1,4 +1,4 @@
-package thirdparty
+package clients
```

Comment:
(nit) How about moving package client files to
 https://github.com/Nexters/myply/tree/main/infrastructure/clients ?